retrolambda/src/main/java/net/orfjackal/retrolambda/lambdas/LambdaNaming.java

```diff
@@ -37,4 +37,12 @@ public static boolean isPlatformFactoryMethod(int access, String name, String de
                 && desc.equals(targetDesc)
                 && Flags.hasFlag(access, ACC_PRIVATE | ACC_STATIC);
     }
+
+    public static boolean isBodyMethodName(String name) {
+        return name.startsWith("lambda\\$");
```

Comment:
`javac` format is "lambda$methodname$number" and ECJ is "lambda$number". I don't think this is ideal, but the name is highly unlikely to change without a major version bump in the JDK which could just as well bring classfile changes that break everything else about this tool.